### PR TITLE
feature: support array of URLs in assets_url interpreter

### PR DESCRIPTION
**Pull request by dkarlovi**

### CODE REVIEW

File: src/ImportDefinitionsBundle/Interpreter/AssetsUrlInterpreter.php [hunk 1]
```diff
@@ -25,12 +25,15 @@ class AssetsUrlInterpreter extends AssetUrlInterpreter
      */
     public function interpret(Concrete $object, $value, Mapping $map, $data, DefinitionInterface $definition, $params, $configuration)
     {
-        $asset = parent::interpret($object, $value, $map, $data, $definition, $params, $configuration);
+        $assets = [];
```
Comment (dpfaffenbauer) [inline src/ImportDefinitionsBundle/Interpreter/AssetsUrlInterpreter.php]: can you also add something like

```
if (is_array($value)) {
for(...)
 ....

return $assetsArray;
}

return [parent::..];
```

just to be BC compatible

Comment (dkarlovi) [reply src/ImportDefinitionsBundle/Interpreter/AssetsUrlInterpreter.php]: But this is BC compatible? You get back either an array of assets or `null`?

Comment (dkarlovi) [reply src/ImportDefinitionsBundle/Interpreter/AssetsUrlInterpreter.php]: Which is exactly what you got before. Or am I missing something?

Comment (dpfaffenbauer) [reply src/ImportDefinitionsBundle/Interpreter/AssetsUrlInterpreter.php]: yes, you are right, I missed the ```(array) $value```

Comment (dkarlovi) [reply src/ImportDefinitionsBundle/Interpreter/AssetsUrlInterpreter.php]: NP. :+1:

**Pull request merged**